src/server/src/main/java/io/cassandrareaper/storage/CassandraStorage.java

```diff
@@ -1824,7 +1824,7 @@ public RetryDecision onWriteTimeout(
       Preconditions.checkState(WriteType.CAS != type ||  ConsistencyLevel.SERIAL == cl);
 
       return null != stmt && !Objects.equals(Boolean.FALSE, stmt.isIdempotent())
-          ? WriteType.CAS == type ? RetryDecision.retry(ConsistencyLevel.ONE) : RetryDecision.retry(cl)
+          ? RetryDecision.retry(cl)
```

Comment:
How does Astra get around all the `CL.LOCAL_ONE` writes otherwise in this class ?

Reply:
We have no other explicit `LOCAL_ONE` writes. I can't remember if the default CL is set at the driver or server level.
I've seen no sign of failed writes aside from failed retries, so I guess Astra sets the default CL server side:
![Capture d’écran 2020-09-15 à 14 57 43](https://user-images.githubusercontent.com/5096002/93213267-ea1fd880-f763-11ea-938f-c9fc05d7a785.png)

Reply:
Weird. I was certain that the driver defaults to `CL.LOCL_ONE`

Reply:
`CassandraStorage` is designed so that the default is `CL.LOCAL_ONE`.

Reply:
references
- https://docs.datastax.com/en/dsbulk/doc/dsbulk/reference/driverOptions.html#driverOptions__datastaxJavaDriverBasicRequestConsistency
- https://docs.datastax.com/en/developer/java-driver/4.9/manual/cloud/